graphql-test-utils/load-schema.js

```diff
@@ -0,0 +1,19 @@
+const fs = require('fs');
```

Comment:
Is this really loading? Is it more `resolveAndCacheSchema`?

Reply:
Couldn't find a better name 😅